### DISCUSSION TITLE
Who's Responsible for Knockoutjs.com? It's time for an update

### DISCUSSION THREAD

**Discussion by mattlacey** [knockout/tko#188] [General]
I feel like in the best interests of the community at large, it would make sense to do a final 4.0 release that drops the beta tag. Those of us using versions tagged alpha are aware that it's perfectly stable in production environments.

Right now there are a lot of JS frameworks moving in the direction of KO with the new hotness that is signals. There are many posts out there saying "this is how KO worked 10 years ago" and all of them tend to link to knockoutjs.com as a result. I believe TKO should see a new release, and the site updated to point to that, or if the legacy site wants to be maintained for posterity (and because hey, KO still works well) at least a banner would go a long way to making the existence of TKO known.

**Comment by itlackey** [comment]
I eventually gave up and moved to svelte. You could also consider htmx.

It's sad, knockout was one of the best JS libraries I've used.
